guide/sapphire/first-command.md

```diff
@@ -0,0 +1,80 @@
+# Your first command
+
+Now that you have created the `commands` folder, you're ready to make your first command file! In Sapphire, we usually define one command per file, but you can also define groups of them. Hop over to your `commands` folder, and create a new file, we'll show how to make a ping command, so we will name this `ping.js`. This is going to be a simple command that replies to a message and then edits with the elapsed time. We'll go into arguments and other features later.
+
+Once you have your file, it's time to get started!
+
+## Creating your command class
+
+Before you do anything, at the start of your file, you're going to need to require Sapphire Framework again, specifically its Command class.
+
+```js
+const { Command } = require('@sapphire/framework');
+```
+
+Commands are classes that are exported in any of the following ways:
+
+```js
+// CommonJS
+module.exports = class MainExportedFoo extends Piece {};
+exports.default = class DefaultExportedFoo extends Piece {};
+exports.myCommand = class CustomExportedFoo extends Piece {};
+
+// ECMAScript Modules
+export default class ExportedDefaultFoo extends Piece {}
+export class ExportedFoo extends Piece {}
+```
```

Comment:
Maybe this should only have the CommonJS way as that is used mostly through the guide and the user might be confused on which to choose?

Reply:
@Danktuary @almostSouji Sapphire's loader supports both CJS and ESM (even mixed within the same project), as well as it supports multiple exports within the same file, which differs from Commando, so I don't really have a good reference at how should we display the possibilities.

Furthermore, all of Sapphire (as well as discord.js) support ESM exports, so the ESM integration here is full.

Any suggestions regarding how to make this less confusing/easier to understand?

Reply:
For now, use the built-in `CodeGroup` and `CodeGroupItem` components: https://vuepress2.netlify.app/reference/default-theme/markdown.html#custom-containers

They're not used anywhere else yet, so see how well it works with ESLint (e.g. test with a missing semicolon). If it doesn't lint, try adding newlines before and after the codeblocks.

Reply:
It lints correctly, and it also works beautifully, thanks!
![image](https://user-images.githubusercontent.com/24852502/126479538-47e166a9-da96-4c52-bcb1-c07317b9b188.png)